src/core/mcpClient.ts

```
@@ -0,0 +1,355 @@
+
+  /**
+   * Create the appropriate transport for the server type
+   */
+  private async createTransport(server: MCPServerConfig) {
+    switch (server.type) {
+      case 'stdio':
+        if (!server.command) {
+          throw new MCPVerificationError('STDIO server missing command', server.name);
+        }
+        
+        const isDebugMode = process.env.DEBUG === '1';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)